catroid/src/androidTest/java/org/catrobat/catroid/uiespresso/activity/rtl/RtlLayoutsTest.java

```diff
@@ -0,0 +1,247 @@
+				.perform(click());
+		onView(withText(R.string.scripts))
+				.check(matches(isCompletelyDisplayed()));
+		onView(withText(R.string.backgrounds))
+				.check(matches(isCompletelyDisplayed()));
+		onView(withText(R.string.sounds))
+				.check(matches(isCompletelyDisplayed()));
+
+		assertTrue(checkTextDirection(getResourcesString(R.string.scripts)));
+		assertTrue(checkTextDirection(getResourcesString(R.string.backgrounds)));
+		assertTrue(checkTextDirection(getResourcesString(R.string.sounds)));
+
+		onView(withText(R.string.backgrounds))
+				.perform(click());
+		openActionBarOverflowOrOptionsMenu(baseActivityTestRule.getActivity());
+		onView(withText(R.string.show_details))
+				.perform(click());
+
+		onView(withIndex(withId(R.id.fragment_look_item_name_text_view), 0))
+				.check(matches(isCompletelyDisplayed()));
+		onView(withIndex(withId(R.id.fragment_look_item_image_view), 0))
+				.check(matches(isCompletelyDisplayed()));
+		onView(withIndex(withId(R.id.fragment_look_item_measure_prefix_text_view), 0))
+				.check(matches(isCompletelyDisplayed()));
+		onView(withIndex(withId(R.id.fragment_look_item_size_prefix_text_view), 0))
+				.check(matches(isCompletelyDisplayed()));
+		onView(withIndex(withId(R.id.fragment_look_item_name_text_view), 0))
+				.check(isLeftOf(withIndex(withId(R.id.fragment_look_item_image_view), 0)));
+		onView(withIndex(withId(R.id.fragment_look_item_measure_prefix_text_view), 0))
+				.check(isLeftOf(withIndex(withId(R.id.fragment_look_item_image_view), 0)));
+		onView(withIndex(withId(R.id.fragment_look_item_size_prefix_text_view), 0))
+				.check(isLeftOf(withIndex(withId(R.id.fragment_look_item_image_view), 0)));
+
+		pressBack();
+		pressBack();
+		pressBack();
+		pressBack();
+		openActionBarOverflowOrOptionsMenu(baseActivityTestRule.getActivity());
+		onView(withText(mainMenuScratchConverterBeta))
+				.perform(click());
+		openActionBarOverflowOrOptionsMenu(baseActivityTestRule.getActivity());
+		onView(withText(R.string.show_details))
+				.perform(click());
+		onView(withIndex(withId(R.id.scratch_projects_list_item_image), 1))
+				.check(matches(isCompletelyDisplayed()));
+		onView(withIndex(withId(R.id.scratch_projects_list_item_title), 1))
+				.check(matches(isCompletelyDisplayed()));
+		onView(withIndex(withId(R.id.scratch_projects_list_item_details_text), 1))
+				.check(matches(isCompletelyDisplayed()));
+		onView(withIndex(withId(R.id.scratch_projects_list_item_title), 1))
+				.check(isLeftOf(withIndex(withId(R.id.scratch_projects_list_item_image), 1)));
+		onView(withIndex(withId(R.id.scratch_projects_list_item_details_text), 1))
+				.check(isLeftOf(withIndex(withId(R.id.scratch_projects_list_item_image), 1)));
+	}
+
+	private boolean checkViewDirection(int viewId) {
+		return ViewCompat.LAYOUT_DIRECTION_RTL == ViewCompat.getLayoutDirection(baseActivityTestRule.getActivity()
+				.findViewById(viewId));
+	}
+
+	private boolean checkTextDirection(String string) {
```

Comment:
ive seen this function with another name already in your other prs, same applies here.

Reply:
done